Bluepill-runner/BluepillRunnerTests/BPReportCollectorTests.m

```diff
@@ -28,27 +28,48 @@ - (void)tearDown {
 }
 
 - (void)testCollectReportsFromPath {
+    NSString *path = [[NSBundle bundleForClass:[self class]] resourcePath];
+    NSString *outputPath = [path stringByAppendingPathComponent:@"result.xml"];
+    [BPReportCollector collectReportsFromPath:path onReportCollected:nil applyXQuery:nil withOutputAtPath:outputPath];
+    NSData *data = [NSData dataWithContentsOfFile:outputPath];
+    NSError *error;
+    NSXMLDocument *doc = [[NSXMLDocument alloc] initWithData:data options:0 error:&error];
+    XCTAssertNil(error);
+    NSArray *testsuitesNodes =  [doc nodesForXPath:[NSString stringWithFormat:@".//%@", @"testsuites"] error:&error];
+    NSXMLElement *root = testsuitesNodes[0];
+    XCTAssertTrue([[[root attributeForName:@"tests"] stringValue] isEqualToString:@"271"], @"test count is wrong");
+    XCTAssertTrue([[[root attributeForName:@"errors"] stringValue] isEqualToString:@"2"], @"test count is wrong");
+    XCTAssertTrue([[[root attributeForName:@"failures"] stringValue] isEqualToString:@"4"], @"test count is wrong");
+
+    NSLog(@"%@, %@, %@", [[root attributeForName:@"tests"] stringValue], [[root attributeForName:@"errors"] stringValue], [[root attributeForName:@"failures"] stringValue]);
```

Comment:
please use BPUtil to print out log

Reply:
This is in a test though... I think `NSLog()` here should be fine.

Reply:
This code already exists in Bluepill, it just moved up a bit when I removed a comment someone else had left there.

Reply:
agree with @ob here.. NSLog() is fine in tests.